site/content/docs/5.2/components/navbar.md

```diff
@@ -279,10 +280,10 @@ Various buttons are supported as part of these navbar forms, too. This is also a
 
 {{< example >}}
 <nav class="navbar navbar-dark bg-dark">
-  <form class="container-fluid justify-content-start align-items-center">
+  <div class="container-fluid justify-content-start align-items-center">
```

Comment:
Not sure about this one but it's more logical to me. Think to change the paragraph above.